internal/app/recordtester/recordtester_app.go

```diff
@@ -156,6 +184,19 @@ func (rt *recordTester) Start(fileName string, testDuration, pauseDuration time.
 	mediaURL := fmt.Sprintf("%s/%s/index.m3u8", ingest.Playback, stream.PlaybackID)
 	glog.V(model.SHORT).Infof("RTMP: %s", rtmpURL)
 	glog.V(model.SHORT).Infof("MEDIA: %s", mediaURL)
+
+	if rt.accessControl && (rt.signingKey != "" && rt.publicKey != "") {
+		token, err := rt.signJwt(stream)
+		if err != nil {
+			return 1, err
+		}
+		mediaURL = fmt.Sprintf("%s?jwt=%s", mediaURL, token)
+		glog.V(model.VERBOSE).Infof("URL with access control for stream id=%s playbackId=%s name=%s mediaURL=%s", stream.ID, stream.PlaybackID, streamName, mediaURL)
+	} else {
+		glog.Warningf("No access control for stream id=%s playbackId=%s name=%s mediaURL=%s", stream.ID, stream.PlaybackID, streamName, mediaURL)
+		return 2, nil
+	}
```

Comment:
No reason to log this as a warning or returning. Executing a test without access control is perfectly acceptable. And we already log above when we use access control so no need to log when we don't.
```suggestion
	}
```